<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblUrl">
          <property name="text">
           <string>URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileUrl">
          <property name="text">
           <string>C:/Users/Mikrolar/Documents/Qt Projects/ReadWriteExcelProj/Book1.xlsx</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblSaveAs">
          <property name="text">
           <string>Save As: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputUrl">
          <property name="text">
           <string>C:/Users/Mikrolar/Documents/Qt Projects/ReadWriteExcelProj/output.xlsx</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string>Row1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Row2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Row3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Col1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Col2</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>i(#$l./;'\</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Hello World</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>9999999999.99</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>free</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="text">
         <string>Read File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWrite">
        <property name="text">
         <string>Write File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
